src/tests/ftest/container/simple_create_delete_test.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/python
-'''
-  (C) Copyright 2017-2019 Intel Corporation.
+"""
+  (C) Copyright 2018-2019 Intel Corporation.
```

Comment:
Why are we changing the copyright timeframe?

Reply:
visual studio linting error.  pydocstyle(D300)  I usually change it to clear out any linting issues before pushing change